### ISSUE TITLE
PvP Loadouts - MOE

### ISSUE THREAD

**Issue by ComboTombo** [ComboTombo/A3-Antistasi#10]
Primary Weapon will be set to the HK G36K (Upper Rail) with an EOTech 552 optic and an AN/PEQ-2A IR laser.

_Standard issue rifle for MOE operatives_

Secondary weapon will be set to the HK USP.

_Standard issue pistol for the Spanish Army_

Uniforms will be set to the Uniform Long Sleeve (Woodland/Arid). 

Vests will be set to the Ciras Land (MTP).

_Although there are woodland & arid vests in FFAA, they're for the infantrymen, specifically the 7th Light Infantry Brigade. This one, however, is dedicated for the MOE (Special Operations Command). This vest is mostly similar to the normal vests in terms of stats._

Each vest will have 2 smoke nades, 2 Alhambra-DO fragmentation nades, 4-5 mags, and a pistol mag.

Helmets will be set to the ACH (Green/Tan).

_This helmet is dedicated to MOE operatives. It has the same stats as the normal infantryman's helmet._

Team Leader gets a GPS and a rangefinder.

_The reason why the team leader gets a rangefinder is that it incentivizes teamwork between the team leader and marksman, or team leader and AT._

NVGs will be set to the AN/PVS-7.

### SPECIAL FOR AT AND AAT ###

AT gets a C90 with a C90 CR-RB HEAT 90mm round loaded and another rocket in a backpack.

_Compared to the C100, this rocket launcher is balanced, provides a challenge, and for the most part works properly (excluding the fact that it's multi-use instead of single-use. The recoilless C100, on the other hand, comes with a rangefinder, built in NVGs, and its rounds have very little to no drop or deviation, in other words a flat trajectory._

AAT gets two C90 CR-RB HEAT 90mm rounds in their backpacks.

_Basically an ammo bearer that carries two more rockets._

### SPECIAL FOR MACHINEGUNNER ###

MG gets an MG4 with an EOTech 552 and an an AN/PEQ-2A IR laser

_It's the standard light machine gun of the Spanish Army._

MG vest gets one 200rnd box.

_The already relatively heavy machinegunner will only get 400 rounds of ammunition._

### SPECIAL FOR MARKSMAN ###

Marksman gets the L96A1 with the Schmidt and Bender 3-12x50 PM II sniper scope

_Standard marksman rifle of the Spanish Army. The two other sniper rifles would be the Barret M95 and the AW 50, and both of these .50 cal snipers only have AP ammo and explosive ammo_

Marksman vest has 10 10rnd magazines.

_It's 110 rounds of ammunition. Surely you can hit SOMETHING with that many bullets..._
